app/src/main/res/xml/locales_config.xml

```diff
@@ -1,10 +1,18 @@
 <?xml version="1.0" encoding="utf-8"?>
+<!--
+ When updating this file also update Utils.localeOptions
+-->
```

Comment:
Is there not a way to retrieve this info at runtime instead of maintaining two different lists?

Reply:
Not that I am aware, but this also maps from locale to its translated name in that language

Reply:
https://stackoverflow.com/questions/74114067/get-list-of-locales-from-locale-config-in-android-13
There is an option here but this requires reading the xml file manually